### ISSUE TITLE
schedule day this week

### ISSUE THREAD

**Issue by whit537** [sproutfund/pghcityoflearning.org#5]
@timothyfcook How about Friday this week? I might have a meeting downtown that morning, and there's the [Code & Supply event](http://www.meetup.com/Pittsburgh-Code-Supply/events/220049911/) at the end of the day (maybe you're going to that?). Would it work for me to be with you at Sprout between, say, 11 and 5?

**Comment by timothyfcook** [comment]
Sounds perfect to me. @MatthewVita probably has work and is busy with his Angular talk prep, but maybe @justinxreese and @cameronscott137 can join us for all or part of that day as well.

How 'bout it, are you two available that day to kick off our work together?
We can take over the back building at Sprout Fund and I will procure fancy lunch (maybe fancy salads and smoothies from Living Juicy?) and buy drinks for all afterwards at Code and Supply party!

**Comment by justinxreese** [comment]
Yep. Friday works for me. May need to run a few errands for the C&S event,
but otherwise free.

**Comment by timothyfcook** [comment]
Awesome. If you want, Justin, you could even come to Sprout Fund a bit earlier than Chad so that I can get you more oriented. Maybe 10:00-30?

**Comment by justinxreese** [comment]
Sounds good

**Comment by MatthewVita** [comment]
day job for me :(
